examples/README.md

```diff
@@ -1268,56 +1269,26 @@ deployment_groups:
          settings:
             key1: $(vars.zone)
             key2: $(resource1.name)
+            # access nested fields
+            key3: $(resource1.nodes[0].private_ip)
+            # arithmetic expression
+            key4: $(vars.num_nodes + 5)
+            # string interpolation expression
+            key5: $("${resource1.name}_${vars.zone}")
```

Comment:
A common use case is people wanting to use a variable inside a script. Is there any way at present to do multiline string interpolation. The equivalent of:

```
contents: |
  #! /bin/bash
  sleep ${var.sleep_count}
```
It looks like the warning for string interpolation is gone. Do we now fully support string interpolation?

I think if any format of multi-line interpolation is possible it would be valuable to show as an example, and if not we should call it out and probably still warn on it.

Reply:
Added support (+doc) for multiline expressions.